### PR TITLE
Io 548

### PR DESCRIPTION
**Pull request by cristain7**

### CODE REVIEW

**Review by jeanettelandoyaune — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "styleguide",
-  "version": "0.99.90",
+  "version": "0.99.91",
```
Comment (larsboldt) [inline package.json:3]: version change should be its own commit

File: src/components/_index.scss [hunk 2]
```diff
@@ -27,7 +27,7 @@
 @use "image/image";
 @use "image_carousel/image_carousel";
 @use "links/link/link";
-@use "links/personalized_content_link/personalized_content_link";
+@use "links/content_link/content_link";
```
Comment (larsboldt) [inline src/components/_index.scss:30]: why name change?

**Review by larsboldt — CHANGES_REQUESTED** [review]
version change should be its own commit

File: src/components/_index.scss (hunk 2 above)
Comment (cristain7) [reply src/components/_index.scss:30]: Forgot to delete this merge request in favor of https://github.com/oslokommune/styleguide_devtools/pull/47

**Comment by cristain7** [comment]
Not merged

**Pull request closed without merging**